FlightReservation/src/consts/style.js

```diff
@@ -66,6 +66,10 @@ const style = StyleSheet.create({
   account_link: {
     color: colors.blue,
   },
+  invalid_input_text: {
+    color: colors.red,
+    // fontWeight: 'bold',
```

Comment:
If you do not occupy the code that is commented, remove it.

Reply:
Deleted.